### PR TITLE
 Deployment of the dke module py-token-fetch [BBPP82-632]

### PR DESCRIPTION
**Pull request by nabilalibou**

### CODE REVIEW

File: deploy/environments/applications.yaml [hunk 1]
```diff
@@ -153,4 +155,4 @@ spack:
     - touchdetector
     - ultraliser
     - unit-test-translator
-    # - zee
+    # - zee
```
Comment (matz-e) [inline deploy/environments/applications.yaml:158]: No newline at end of file?
```suggestion
    # - zee
```

**Review by matz-e — COMMENTED** [review]
Thank you for the new package & module!

Looks good apart from one nitpick.

File: deploy/environments/applications.yaml (hunk 1 above)
Comment (nabilalibou) [reply deploy/environments/applications.yaml:158]: @matz-e Thank you for your corrections !

Comment (matz-e) [reply deploy/environments/applications.yaml:158]: @Nabil-AL you're welcome! Please note that I've also upgraded @BlueBrain/dke 's permissions, you should be able to push branches here and merge pull requests (we let you merge the PRs you open after approval)

Comment (nabilalibou) [reply deploy/environments/applications.yaml:158]: Thank you, by the way I cannot commit your suggestion as github considers that no changes were made to the code

Comment (matz-e) [reply deploy/environments/applications.yaml:158]: Sigh, fair enough. The next edit to the file will probably restore that line ending.

**Review by matz-e — APPROVED** [review]

**Pull request merged**